scripts/tests

```diff
@@ -934,6 +942,13 @@ def cli():
         "all", help="Run all tests (except benchmark for now)", parents=[common_parser]
     )
 
+    all_parser.add_argument(
+        "--check-namespace",
+        help="Check namespacing of binaries",
+        action="store_true",
+        default=False,
+    )
+
```

Comment:
Why is this argument added to both the `all_parser` and the `func_parser`? 
The commit message suggests it should only be present for the `func` tests?

Reply:
Func can be run through `tests all`